snippets/visualbasic/VS_Snippets_Wpf/HOWTOApplicationModelSnippets/visualbasic/access-by-line.txt

```diff
@@ -1 +1,2 @@
-﻿App.xaml.vb: ~/docs/framework/wpf/app-development/application-management-overview.md
+﻿Application.xaml.vb: ~/docs/framework/wpf/app-development/application-management-overview.md
```

Comment:
```suggestion
application.xaml.vb: ~/docs/framework/wpf/app-development/application-management-overview.md
```

Reply:
should we perhaps link to the file on GitHub for easier access?

```suggestion
application.xaml.vb: https://github.com/dotnet/docs/blob/master/docs/framework/wpf/app-development/application-management-overview.md
```

Reply:
The link is a good idea, @mairaw. I'll modify the other files in a separate PR to include links.